File: 0258. Add Digits.py
class Solution:
    def addDigits(self, num: int) -> int:
        while num >= 10 :
            n = num
            sum = 0
           
            while int(n/10) > 0:
                sum += int(n%10)
                n = int(n/10) 
            
            num = int(n%10) + sum
            
        
        return num

File: 0258. Add Digits.cs
public class Solution {
    public int AddDigits(int num) {
        while(num >= 10){
            int n = num;
            int sum = 0;
       
            while(n/10 > 0){
                sum += n%10;
                n = n/10;  
            }
       
            num = n%10 + sum;
        }
        
        return num;
    }
}